chainer/testing/backend.py

```diff
@@ -83,7 +83,9 @@ def device(self):
         return self._device
 
     def __enter__(self):
-        self._contexts = [
+        if self._contexts is None:
```

Comment:
I think we can skip this check since it's initialized to a non-`None` in `__init__`.

Reply:
Fixed.